hydra-configs-torchvision/hydra_configs/torchvision/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+#
+# Generated by configen, do not edit.
+# See https://github.com/facebookresearch/hydra/tree/master/tools/configen
+# fmt: off
+# isort:skip_file
+# flake8: noqa
+
+from packaging import version
+from pkg_resources import get_distribution
+import warnings
+import torchvision
+
+CONFIGS_VERSION = get_distribution('hydra-configs-torchvision').version
```

Comment:
not sure I understand, is the config version by definition the version that is currently installed at runtime?

Reply:
the idea here is to check if the currently installed version of `torchvision` matches the currently installed version of `hydra-configs-torchvision`. If your current environment has a mismatch in these, this will warn you.

Reply:
roger.